.drone.jsonnet

```diff
@@ -11,17 +11,27 @@ local server_ref_map = {
 };
 
 local builddir = 'verylongdirnameforverystrangecpackbehavior';
-local cmakeflags = '-DCMAKE_BUILD_TYPE=RelWithDebInfo -DPLUGIN_COLUMNSTORE=YES -DPLUGIN_XPAND=NO -DPLUGIN_MROONGA=NO -DPLUGIN_ROCKSDB=NO -DPLUGIN_TOKUDB=NO -DPLUGIN_CONNECT=NO -DPLUGIN_SPIDER=NO -DPLUGIN_OQGRAPH=NO -DPLUGIN_SPHINX=NO';
+
+// This is stupid use of PLUGIN_*=NO syntax. It should just skip building
```

Comment:
do you know the proper path how to reach that? it looks stupid indeed. but I don't know other ways to skip plugins. you could address that to mariadb server project. debian/autobase-deb.sh uses the same stupid way.

Reply:
For autobake-deb.sh it is not stupid, since it wants to build as much of the server as possible and "blacklist" only a few ones that take too long.

In ColumnStore you want to avoid building anything, and only "whitelist" ColumnStore to be built.